src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt

```diff
@@ -0,0 +1,18 @@
+package hs.kr.equus.user.global.utils.pass
+
+import hs.kr.equus.user.domain.auth.exception.InvalidUrlException
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class RedirectUrlChecker {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line